Tests/LayoutTests/LayoutItemTests.swift

```diff
@@ -61,45 +61,45 @@ final class LayoutItemTests: XCTestCase {
 
     // MARK: - Size
 
-    func testSizeWidthHeightPriority() {
+    func testSizeWithSizePriority() {
```

Comment:
I believe this test name should be:

```suggestion
    func testSizePriority() {
```

Reply:
I realize we are just reusing the old name but this may have been one that was missed as we were deciding how and when to use `With` in the test naming conventions doc